utils/create_validate_pr_quickstarts.ts

```diff
@@ -53,17 +53,41 @@ const main = async () => {
       return acc;
     }, []);
 
+  const invalidQuickstarts = quickstarts
+    .map((qs) => {
+      qs.validate();
+      if (qs.isValid) {
+        return qs;
+      } else {
+        return undefined;
+      }
+    })
+    .filter(Boolean);
```

Comment:
This could be simplified down to 
```js
const invalidQuickstarts = quickstarts
  .map((qs) => qs.validate())
  .filter((qs) => !qs.isValid);
```
However, that does make us loop through the array twice, so totally up to you which direction you'd like to go.

Reply:
I tried this approach however `.validate()` doesn't return anything, so `.filter()` receives nothing.

I went with the first approach, just a little cleaner instead